docker-containers.el

```diff
@@ -286,6 +295,12 @@ Remove the volumes associated with the container when VOLUMES is set."
               (?v "Volumes" "-v"))
   :actions  '((?D "Remove" docker-containers-rm-selection)))
 
+(docker-utils-define-popup docker-containers-kill-popup
+  "Popup for kill signaling containers"
+  'docker-containers-popups
+  :man-page "docker-kill"
+  :actions  '((?K "kill" docker-containers-kill-selection)))
+
```

Comment:
Adding the `SIGNAL` option is pretty easy, can you add it? `:options '((?s "Signal" "-s "))` 

Also the docstring is a bit weird.